### ISSUE TITLE
Can't use installed locales

### ISSUE THREAD

**Issue by Rudloff** [heroku/heroku-buildpack-locale#14]
Hello,

I need some locales in my app so I added [a .locales file](https://github.com/Rudloff/alltube/blob/master/.locales) in my repository and redeployed my app.
I can see that the corresponding Ubuntu packages are installed correctly but when I run `locale -a` in the dyno, it only returns `en_*` locales:
```
C
C.UTF-8
en_AG
en_AG.utf8
en_AU.utf8
en_BW.utf8
en_CA.utf8
en_DK.utf8
en_GB.utf8
en_HK.utf8
en_IE.utf8
en_IL
en_IL.utf8
en_IN
en_IN.utf8
en_NG
en_NG.utf8
en_NZ.utf8
en_PH.utf8
en_SG.utf8
en_US.utf8
en_ZA.utf8
en_ZM
en_ZM.utf8
en_ZW.utf8
POSIX
```

Am I missing something?

Here is the full build log: https://gist.github.com/Rudloff/a49a8cc576ebaa83e949bf7e89ec2bcd

**Comment by argentinaluiz** [comment]
Me too

**Comment by Mygod** [comment]
Same here. I also tried `heroku run ". /app/.profile.d/000_locale.sh && locale -a"` but got the same results.

**Comment by Mygod** [comment]
Deprecates cedar-14 on May 1 while refusing to fix a three year old bug on the new platform. Good job guys @heroku.

**Comment by evdoks** [comment]
I can use the installed locales within my python application. E.g., 
```python
import locale
locale.setlocale(locale.LC_ALL, "de_DE.UTF-8")
```
starts working when using this buildpack.

However, when I try to set system locale using `LANG` config variable, the setting seems to be ignored. Also, running `locale -a` does not list `de_DE.UTF-8`, giving the same list as @Rudloff pasted.

**Comment by bemoty** [comment]
Over a year after the creation of this issue, I'm experience exactly the same. Lang buildpack outputs that it has installed the locale, `locale -a` does not output the locale.

<pre>
remote: -----> Locale app detected
remote: === Updating or installing language-pack-de-base
remote: Hit:1 http://archive.ubuntu.com/ubuntu bionic InRelease
remote: Hit:2 http://apt.postgresql.org/pub/repos/apt bionic-pgdg InRelease
remote: Hit:3 http://archive.ubuntu.com/ubuntu bionic-security InRelease
remote: Hit:4 http://archive.ubuntu.com/ubuntu bionic-updates InRelease
remote: Reading package lists...
remote: Reading package lists...
remote: Building dependency tree...
</pre>

Python (obviously) can't find the locale either.

<pre>
2020-10-07T07:02:19.085677+00:00 app[web.1]:   File "/app/app.py", line 21, in <module>
2020-10-07T07:02:19.085677+00:00 app[web.1]:     locale="German"
2020-10-07T07:02:19.085678+00:00 app[web.1]:   File "/app/.heroku/python/lib/python3.7/locale.py", line 608, in setlocale
2020-10-07T07:02:19.085678+00:00 app[web.1]:     return _setlocale(category, locale)
2020-10-07T07:02:19.085678+00:00 app[web.1]: locale.Error: unsupported locale setting
</pre>

If there really is nothing I, or we, are doing wrong, this issue makes the buildpack effectively useless. Why has no maintainer looked into this for over a year? Last activity was 20 days ago. @Rudloff have you found a solution in the meantime? 😬

**Comment by Rudloff** [comment]
No, I could not find a solution. (But I don't need this for my app anymore.)
